src/components/Table/TableComponents/TableHeader.tsx

```diff
@@ -128,9 +128,14 @@ export const TableHeader = <T extends Record<string, any>>({
 		<thead>
 			<tr>
 				{canDrag && (
-					<td className="neo-table__dnd-th">
-						<div>&nbsp;</div>
-					</td>
+					<th className="neo-table__dnd-th">
+						<div
+							role="button"
+							aria-label={translations.dragHandle || "Drag handle"}
```

Comment:
add `dragHandle` default text to the file `src/components/Table/helpers/default-data.ts` and then remove this `||`.
```suggestion
							aria-label={translations.dragHandle}
```

Reply:
great!